test/test_quantized_tensor.py

```diff
@@ -60,7 +62,7 @@ def test_qtensor_quant_dequant(self):
         zero_point = 2
         qr = torch.quantize_linear(r, scale, zero_point, torch.quint8)
         rqr = qr.dequantize()
-        self.assertTrue(np.allclose(r.numpy(), rqr.numpy(), atol=2 / scale))
+        self.assertAlmostEqual(r.numpy(), rqr.numpy(), delta=2 / scale)
```

Comment:
we probably don't need to convert to numpy in this case, same for the other places.